apps/condo/domains/analytics/components/Dashboard.tsx

```diff
@@ -0,0 +1,489 @@
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'center',
+}
+
+type DataCardProps = { label: string, value: string | number, secondaryLabel?: string }
+
+const DataCard: React.FC<DataCardProps> = ({ label, value, secondaryLabel }) => (
+    <Col>
+        <Space direction='vertical' align='center' size={8}>
+            <Typography.Title level={3} type='primary'>{value}</Typography.Title>
+            <div style={DATA_CARD_DESCRIPTION_CONTAINER_STYLE}>
+                <Typography.Text type='primary'>{label}</Typography.Text>
+                {secondaryLabel && (
+                    <Typography.Text type='secondary' size='small'>{secondaryLabel}</Typography.Text>
+                )}
+            </div>
+        </Space>
+    </Col>
+)
+
+const PerformanceCard = ({ organizationId, paymentSum, receiptSum, residentsData, paymentLoading }) => {
+    const intl = useIntl()
+    const SummaryTitle = intl.formatMessage({ id: 'pages.reports.summary' })
+    const DoneLabel = intl.formatMessage({ id: 'Done' })
+    const InWorkLabel = intl.formatMessage({ id: 'ticket.status.IN_PROGRESS.name' })
+    const NewTicketsLabel = intl.formatMessage({ id: 'ticket.status.OPEN.many' })
+    const ClosedTicketsLabel = intl.formatMessage({ id: 'ticket.status.CLOSED.many' })
+
+    const PaymentsAmountPercent = intl.formatMessage({ id: 'pages.reports.paymentsAmountPercent' })
+    const PaymentsAmount = intl.formatMessage({ id: 'pages.reports.paymentsAmount' })
+    const ResidentsInApp = intl.formatMessage({ id: 'pages.reports.residentsWithApp' })
+    const TodayTitle = intl.formatMessage({ id: 'pages.reports.aggregatePeriod.today' })
+    const MonthTitle = intl.formatMessage({ id: 'pages.reports.aggregatePeriod.month' })
+    const TotalTitle = intl.formatMessage({ id: 'pages.reports.aggregatePeriod.total' })
+
+    const [completionPercent, setCompletionPercent] = useState('—')
+    const [paymentsAmountPercent, setPaymentsAmountPercent] = useState('—')
+    const [residentsCount, setResidentsCount] = useState(0)
+    const ticketCounts = useRef(null)
+
+    const [loadTicketCounts, { loading: ticketCountLoading }] = useLazyQuery(GET_TICKETS_COUNT_QUERY, {
+        onCompleted: (result) => {
+            ticketCounts.current = result
+        },
+        fetchPolicy: 'cache-first',
+    })
+    const [loadMonthTicketCounts, { loading: monthTicketLoading }] = useLazyQuery(GET_TICKETS_COUNT_QUERY, {
+        onCompleted: (result) => {
+            if (result.all.count > 0) {
+                setCompletionPercent((result.completed.count + result.closed.count / result.all.count * 100)
+                    .toFixed(0) + '%'
+                )
+            }
+        },
+        fetchPolicy: 'cache-first',
+    })
+
+    useEffect(() => {
+        const ticketWhere = {
+            organization: { id: organizationId },
+            createdAt_gte: dayjs().startOf('day').toISOString(),
```

Comment:
I don’t know how optimal dayJs works, but it might be worth at the beginning of the hook const now = dayJs() and then use this object below.